### ISSUE TITLE
No dead links

### ISSUE THREAD

**Issue by ajbraus** [cmaher92/ConnectAll#46]
There can be no dead links.

**Comment by lesliekimm** [comment]
yes, all links will go to an actual tutorial for the presentation

**Comment by extrajordanary** [comment]
unless there is a specific known dead link to remove, this ticket should be closed. it's more of a piece of generic advice, not an actionable task

**Comment by cmaher92** [comment]
@extrajordanary There are dead links on the home page that will be addressed once we begin manually adding tutorials. I will keep this ticket open & updated until all dead links are removed from the home page.
